.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "/Users/mahmoud/src/sbxscripting/bazelbuild/starlarky/.tmp/bazel"]
+	path = /Users/mahmoud/src/sbxscripting/bazelbuild/starlarky/.tmp/bazel
```

Comment:
`mahmoud` ?

Reply:
@mahmoudimus

Reply:
Oh hmm, yes, I need to keep this relative path, instead of absolute. Does that work?

Reply:
yeap, that should work

Reply:
I will be merging this, can you please fix it in #5?